fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1039,6 +1040,32 @@ public void testIngestOnSubtree() throws Exception {
 
     }
 
+    /**
+     * @author griffinj
+     * Ensure that the objects cannot be pairtree child resources
+     * @see <a href="https://jira.duraspace.org/browse/FCREPO-1505">FCREPO-1505</a>
+     *
+     */
+    @Test
+    public void testIngestOnPairtree() throws Exception {
+        HttpPost method = postObjMethod("");
+        HttpResponse response = client.execute(method);
+        final String location = response.getFirstHeader("Location").getValue();
+
+        // Get the segments from the URL
+        final String[] segments = location.split("/");
+
+        LOGGER.info("POSTing to " + serverAddress + "/" + segments[4] + "/");
+
+        // Attempt to POST to the child of the pairtree resource...
+        method = new HttpPost(serverAddress + segments[3] + "/" + segments[4] + "/");
+        response = client.execute(method);
+
+        // ...which should be a forbidden operation
+        final int status = response.getStatusLine().getStatusCode();
+        assertEquals("Created a child Object for a pairtree child node", FORBIDDEN.getStatusCode(), status);
```

Comment:
This message could be a bit more clear, e.g. "Created a child Object under a pairtree node, which should not be allowed."